### PR TITLE
Creation of the Navigation bar

### PR DESCRIPTION
**Pull request by vado21**
Team! I help Imani with the develop of the navigation bar and I modify the 3 files.

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -22,38 +22,51 @@
     padding: 0;
 }
 
-body {
-    font-family: "Arial", Helvetica, sans-serif;
-}
+body {}
```
Comment (kassidywebb) [inline src/main/webapp/css/main.css]: Do you need this?

File: src/main/webapp/event.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        </ul>
+    </nav>
+    <div class="content">
+
+        <div class="title">
+            <p id="event-title"> Event Title</p>
+        </div>
+
+        <div class="details">
+            <p id="username" class="username" style="font-family:Nunito">Author:</p>
```
Comment (kassidywebb) [inline src/main/webapp/event.html]: Do you need in-line css or can that be put in the css file?

**Review by kassidywebb — CHANGES_REQUESTED** [review]

File: src/main/webapp/css/main.css (hunk 1 above)
Comment (vado21) [reply src/main/webapp/css/main.css]: No sorry! I forgot to change it.

File: src/main/webapp/event.html (hunk 2 above)
Comment (vado21) [reply src/main/webapp/event.html]: Sorry, no!

File: src/main/webapp/js/navigation-loader.js [hunk 3]
```diff
@@ -31,26 +31,37 @@ function addLoginOrLogoutLinkToNavigation() {
         })
         .then((loginStatus) => {
             if (loginStatus.isLoggedIn) {
-                navigationElement.appendChild(createListItemUl(createListItem(createLink(
-                    '/user-page.html?user=' + loginStatus.username, 'Your Page'))));
+                /**Creation of the menu and each element of the navigation bar **/
+                navigationElement.appendChild(createListItemUl(createLink(
+                    '/user-page.html?user=' + loginStatus.username, createListItem('Profile'))));
```
Comment (ryanluo) [inline src/main/webapp/js/navigation-loader.js:36]: Note: if we change the paths in the backend, these need to be changed too. Can you make a comment in each of the views that navigation-loader.js is a dependency?

File: src/main/webapp/event.html (hunk 2 above)
Comment (ryanluo) [reply src/main/webapp/event.html]: +1, let's put it in the css file.

**Review by ryanluo — CHANGES_REQUESTED** [review]
Why was home.css deleted? We jus thave a new Css file now?

File: src/main/webapp/css/main.css (hunk 1 above)
Comment (vado21) [reply src/main/webapp/css/main.css]: Done!

File: src/main/webapp/event.html (hunk 2 above)
Comment (vado21) [reply src/main/webapp/event.html]: Done!

File: src/main/webapp/js/navigation-loader.js (hunk 3 above)
Comment (vado21) [reply src/main/webapp/js/navigation-loader.js:36]: Sorry ryan I forgot to change this

Comment (ryanluo) [reply src/main/webapp/js/navigation-loader.js:36]: did you make a comment in the views?

**Review by ryanluo — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 4]
```diff
@@ -294,7 +294,7 @@ public void saveEventInformation(List<Event> events, PreparedQuery results) {
 
   /**This function gives a Eventby reference given a query and a empty Event as a parameter**/
   
-  public void saveIndividualEvent(Event event,PreparedQuery results) {
+  public Event saveIndividualEvent(PreparedQuery results) {
```
Comment (ryanluo) [inline src/main/java/com/google/codeu/data/Datastore.java]: Why is this included in this pull request?

**Review by ryanluo — CHANGES_REQUESTED** [review]

**Review by ryanluo — APPROVED** [review]
